ci/ci/build.py

```diff
@@ -88,22 +88,27 @@ def build(self, batch, code, scope):
             if step.scopes is None or scope in step.scopes:
                 step.build(batch, code, scope)
 
-        parents = set()
+        names_to_jobs = {}
         for step in self.steps:
-            parents.update(step.wrapped_job())
-        parents = list(parents)
+            wrapped = step.wrapped_job()
+            if wrapped:
```

Comment:
When would wrapped be None or empty?

Reply:
It is empty if you have not run `build` yet on that step.